feat(connector): Add support for `execute` procedure in JDBC connectors (#27282)
prestodb/presto: presto-docs/src/main/sphinx/connector/mysql.rst

No other types are supported.

Procedures
----------

Use the :doc:`/sql/call` statement to perform data manipulation or administrative tasks. Procedures are available in the ``system`` schema of the catalog.

Execute Procedure
^^^^^^^^^^^^^^^^^

> Underlying datasource may support some operation or sql syntax which is not supported by Presto, either at the parser level or at the connector level.

Reviewer: ```suggestion
Underlying datasources may support some operation or SQL syntax which is not supported by Presto, either at the parser level or at the connector level.
```

> Trying to run such SQL statement via Presto can result in errors during parsing or analysing. For example, Mysql database supports creating AUTO_INCREMENT

Reviewer: ```suggestion
Trying to run such SQL statements in Presto can result in errors during parsing or analysing. For example, MySQL supports creating AUTO_INCREMENT
```

> primary keys which is not supported via presto. Running this procedure enables users to do a SQL passthrough to the underlying database and presto just acts

Reviewer: ```suggestion
primary keys which is not supported in Presto. Running this procedure enables users to do a SQL passthrough to the underlying database and presto just acts
```

as a middle man for passing the statement.

The following arguments are available:

============= ========== =============== =======================================================================
Argument Name Required   Type            Description
============= ========== =============== =======================================================================

> ``QUERY``     No         string          Sql statement to run

Reviewer 2: I suppose `QUERY` is a mandatory parameter?

Author: Actually yes, I wanted to say using named argument is not mandatory, we can directly pass the sql statement in the procedure call. But the examples cover that already. 
Let me mark it as Yes.

> ``QUERY``     Yes        string          Sql statement to run

Reviewer: ```suggestion
``QUERY``     Yes        string          SQL statement to run
```
